### ISSUE TITLE
how can I print run_pretraining.py loss?

### ISSUE THREAD

**Issue by antxiaojun** [google-research/bert#106]
INFO:tensorflow:global_step/sec: 9.84722
INFO:tensorflow:examples/sec: 315.111
INFO:tensorflow:global_step/sec: 9.84566
INFO:tensorflow:examples/sec: 315.061
INFO:tensorflow:global_step/sec: 9.84708
INFO:tensorflow:examples/sec: 315.107
INFO:tensorflow:global_step/sec: 9.85038
INFO:tensorflow:examples/sec: 315.212
INFO:tensorflow:global_step/sec: 9.84114
INFO:tensorflow:examples/sec: 314.917
INFO:tensorflow:global_step/sec: 9.84841
INFO:tensorflow:examples/sec: 315.149

**Comment by antxiaojun** [comment]
W1112 17:29:57.142938 Reloader tf_logging.py:120] Found more than one graph event per run, or there was a metagraph containing a graph_def, as well as one or more graph events.  Overwriting the graph with the newest event.
W1112 17:29:57.142938 139873032353536 tf_logging.py:120] Found more than one graph event per run, or there was a metagraph containing a graph_def, as well as one or more graph events.  Overwriting the graph with the newest event.
W1112 17:29:57.143829 Reloader tf_logging.py:120] Found more than one metagraph event per run. Overwriting the metagraph with the newest event.
W1112 17:29:57.143829 139873032353536 tf_logging.py:120] Found more than one metagraph event per run. Overwriting the metagraph with the newest event.
W1112 17:29:57.144825 Reloader tf_logging.py:120] Found more than one graph event per run, or there was a metagraph containing a graph_def, as well as one or more graph events.  Overwriting the graph with the newest event.
W1112 17:29:57.144825 139873032353536 tf_logging.py:120] Found more than one graph event per run, or there was a metagraph containing a graph_def, as well as one or more graph events.  Overwriting the graph with the newest event.
W1112 17:29:57.145925 Reloader tf_logging.py:120] Found more than one metagraph event per run. Overwriting the metagraph with the newest event.
W1112 17:29:57.145925 139873032353536 tf_logging.py:120] Found more than one metagraph event per run. Overwriting the metagraph with the newest event.
E1112 17:29:57.177515 MainThread program.py:201] Tensorboard could not bind to unsupported address family ::
E1112 17:29:57.177515 139879789889344 program.py:201] Tensorboard could not bind to unsupported address family ::
ERROR: Tensorboard could not bind to unsupported address family ::

**Comment by xwzhong** [comment]
you shuld learn how to debug

**Comment by jacobdevlin-google** [comment]
Someone had suggested this:

```
logging_hook = tf.train.LoggingTensorHook({"loss": total_loss}, every_n_iter=10)
output_spec = tf.contrib.tpu.TPUEstimatorSpec(
mode=mode,
loss=total_loss,
train_op=train_op,
training_hooks=[logging_hook],
scaffold_fn=scaffold_fn)
```

This is very noisy though since it just prints the training loss from an arbitrary batch.

Or you can run an eval on your `output_dir` in a separate terminal.

**Comment by razrLeLe** [comment]
@antxiaojun It looks like tensorboard is trying to use ipv6 address, which may not be supported, you can specify a host, like '--host example.com' to avoid this error.
